x/crosschain/genesis.go

```diff
@@ -10,7 +11,9 @@ import (
 // InitGenesis initializes the crosschain module's state from a provided genesis
 // state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
-	k.SetZetaAccounting(ctx, genState.ZetaAccounting)
+	// Always set the zeta accounting to zero at genesis.
+	// ZetaAccounting value is build by iterating through all the cctxs and adding the amount to the zeta accounting.
+	k.SetZetaAccounting(ctx, types.ZetaAccounting{AbortedZetaAmount: sdkmath.ZeroUint()})
```

Comment:
https://github.com/zeta-chain/zeta-node/blob/b2c1b3220a0890aa644f3a85d6629c8ed93e186e/x/crosschain/genesis.go#L93-L96

The export is not needed anymore, but it is still fine to keep it as it is .

Reply:
Q: so the given `genState` is empty in its cctx list and that is why we set ZetaAccounting to zero?

Reply:
Not sure I understand the question , 
ZetaAccounting only accounts for the aborted zeta amount right now , which is calculated by iterating over cctxs. 
We do not need to set it manually.